brave-bin/PKGBUILD

```diff
@@ -55,7 +53,10 @@ package() {
 
 	install -Dm0755 "$pkgname.sh" "$pkgdir/usr/bin/brave"
 	install -Dm0644 -t "$pkgdir/usr/share/applications/" "brave-browser.desktop"
-	install -Dm0644 "logo.png" "$pkgdir/usr/share/pixmaps/brave-desktop.png"
+	# Install the icon using files in the archive
```

Comment:
```suggestion
```

Code comments are good, but what is going on here is pretty obvious. lets just get down to business.

Reply:
I've removed the super obvious comment 😄